src/main/kotlin/com/quipalup/katydid/logentry/primaryadapter/httprest/LogEntryResource.kt

```diff
@@ -19,9 +21,10 @@ data class LogEntryResourceAttributes(
 )
 
 data class LogEntryRequestDocument(
-    val data: LogEntryRequestResource
+    val data: CreateLogEntryByFieldCommand
```

Comment:
```suggestion
    val data: LogEntryRequestResource
```
That was ok as it was. `LogEntryRequestDocument` and `LogEntryRequestResource` are classes to capture JSON structure in network layer. In application layer we have the agnostic types of our application that are independent of any communication network standard or protocol.